v0x04/match.py

```diff
@@ -43,11 +43,11 @@ def match13_no_strict(flow_to_install, stored_flow_dict):
         return False
     if not _match_table_id(flow_to_install, stored_flow_dict):
         return False
-    if "match" not in flow_to_install or "match" not in stored_flow_dict:
+    if "match" not in flow_to_install:
         return stored_flow_dict
     if not flow_to_install["match"]:
         return stored_flow_dict
-    if len(flow_to_install["match"]) > len(stored_flow_dict["match"]):
+    if "match" not in stored_flow_dict or (len(flow_to_install["match"]) > len(stored_flow_dict["match"])):
```

Comment:
@Alopalao, looks like we can get rid of the first clause of this or `"match" not in stored_flow_dict`. 

Rationale: Up to this conditional here, `len(flow_to_install["match"])` is always expected to be > 0, since we have that prior `if "match" not in flow_to_install` early return returning the matched `stored_flow_dict`. And, in a case where the `stored_flow_dict["match"]` is empty then `1 > 0` it'll enter the if returning False as we'd expect.

Can you double check this?

Reply:
Reduced to `(len(flow_to_install["match"]) > len(stored_flow_dict.get("match", [])))` to avoid `KeyError`

Reply:
Great.